### PR TITLE
flake8 fixes

### PR DESCRIPTION
**Pull request by jaheba**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

**Please tag this pr with at least one of these labels to make our release process faster:** BREAKING, new feature, bug fix, other change, dev setup

### CODE REVIEW

File: src/gluonts/model/deepar/_network.py [hunk 1]
```diff
@@ -1102,9 +1102,7 @@ def sampling_decoder(
             repeated_past_target = F.concat(
                 repeated_past_target, new_samples, dim=1
             )
-            repeated_past_observed_values = F.concat(
-                repeated_past_target, F.ones_like(new_samples), dim=1
-            )
+
```
Comment (jaheba) [inline src/gluonts/model/deepar/_network.py:1105]: Was this ever intended to be used?

File: src/gluonts/nursery/autogluon_tabular/predictor.py [hunk 2]
```diff
@@ -75,8 +74,6 @@ def get_features_dataframe(
     assert past_data is None or series.index.freq == past_data.index.freq
     assert past_data is None or series.index[0] > past_data.index[-1]
 
-    cal = calendar()
```
Comment (jaheba) [inline src/gluonts/nursery/autogluon_tabular/predictor.py:78]: Was this ever intended to be used?

File: src/gluonts/nursery/spliced_binned_pareto/training_functions.py [hunk 3]
```diff
@@ -82,11 +82,6 @@ def eval_on_series(
         unfold_layer = torch.nn.Unfold(
             kernel_size=(1, window_length), stride=stride
         )
-        fold_layer = torch.nn.Fold(
```
Comment (jaheba) [inline src/gluonts/nursery/spliced_binned_pareto/training_functions.py:85]: Was this ever intended to be used?

File: src/gluonts/nursery/spliced_binned_pareto/training_functions.py [hunk 4]
```diff
@@ -256,7 +252,6 @@ def highlight_min(data, color="lightgreen"):
         is_min = data == data.min()
         return [attr if v else "" for v in is_min]
     else:  # from .apply(axis=None)
-        is_max = data == data.min().min()
```
Comment (jaheba) [inline src/gluonts/nursery/spliced_binned_pareto/training_functions.py:259]: Was this ever intended to be used?

File: src/gluonts/mx/linalg_util.py [hunk 5]
```diff
@@ -207,7 +207,7 @@ def jitter_cholesky(
             # TODO: Add support for symbolic case: Cannot use <= operator with symbolic variables
             assert F.abs(L.nansum() - L.sum()) <= 1e-1
             return L
-        except:
+        except Exception:  # TODO: this looks weird
```
[the remark this answers is not in the document]
Comment (jaheba) [reply src/gluonts/mx/linalg_util.py:210]: Probably, if I just knew which one.

**Review by lostella — APPROVED** [review]
